### PR TITLE
Docs: Update canvas docs for 10.2

### PR DESCRIPTION
**Pull request by nmarrs**
Update canvas docs for 10.2

Also update convention of "enables" to "lets"

Related changes are:
https://github.com/grafana/grafana/pull/74389
https://github.com/grafana/grafana/issues/74149

### CODE REVIEW

File: docs/sources/panels-visualizations/visualizations/canvas/index.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ The text element enables you to easily add text to the canvas. The element also
 
 {{< video-embed src="/static/img/docs/canvas-panel/canvas-text-9-2-0.mp4" max-width="750px" caption="Text element demo" >}}
 
+### Ellipse
+
+The ellipse element enables you to add a basic ellipse to the canvas. Ellipse elements support displaying text (both fixed and field data) as well as can change background color based on data thresholds.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/canvas/index.md:44]: Do you mean the background color of the ellipse element? I've edited assuming that meaning.
```suggestion
The ellipse element lets you add a basic ellipse to the canvas. An ellipse element can display text (both fixed and field data) and its background color can be changed based on data thresholds.
```

File: docs/sources/panels-visualizations/visualizations/canvas/index.md [hunk 2]
```diff
@@ -53,6 +57,16 @@ The server element enables you to easily represent a single server, a stack of s
 
 {{< figure src="/media/docs/grafana/canvas-server-element-9-4-0.png" max-width="750px" caption="Canvas server element" >}}
 
+### Button
+
+The button element enables you to add a basic button to the canvas. Button elements support triggering basic, unauthenticated API calls. API settings are found in the button element editor. You can pass template variables in the API editor.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/canvas/index.md:62]:
```suggestion
The button element lets you add a basic button to the canvas. Button elements support triggering basic, unauthenticated API calls. API settings are found in the button element editor. You can also pass template variables in the API editor.
```

File: docs/sources/panels-visualizations/visualizations/canvas/index.md [hunk 3]
```diff
@@ -53,6 +57,16 @@ The server element enables you to easily represent a single server, a stack of s
...
+
+{{% admonition type="note" %}}
+To trigger an API call when clicking on a button in the canvas, [inline editing](#inline-editing) must be disabled.
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/visualizations/canvas/index.md:65]:
```suggestion
For a button click in a canvas to trigger an API call, [inline editing](#inline-editing) must be disabled.
```

**Review by imatwawana — COMMENTED** [review]
Mostly small wording things, but I had one spot I was unsure about.

File: docs/sources/panels-visualizations/visualizations/canvas/index.md (hunk 3 above)
Comment (nmarrs) [reply docs/sources/panels-visualizations/visualizations/canvas/index.md:65]: hm the wording here feels a little off - I will try to simplify this a bit

**Review by imatwawana — APPROVED** [review]

**Pull request merged**